sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool for recovering MBR partition tables in hard disks"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2017 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="23b651d1e9c6ae1454e5d3fe315d7d4e145661c0"
```

Comment:
Your recipe is version 0.3, but you're building from HEAD. Is there any reason not to use the [latest release tag][1]?

[1]: https://github.com/baruch/gpart/releases/tag/0.3

Reply:
No,because contains necessary patch

Reply:
@fbrosson should this recipe version be `0.3~git` instead? Given that it's not using the [latest release tag] upstream

[0]: https://github.com/baruch/gpart/releases/tag/0.3